src/Files.App/Filesystem/FilesystemOperations/FileSystemProgress.cs

```diff
@@ -1,100 +1,104 @@
 ﻿// Copyright (c) 2023 Files Community
 // Licensed under the MIT License. See the LICENSE.
 
-using Files.App.Helpers;
-using Files.Shared.Enums;
-using System;
-
 namespace Files.App.Filesystem
 {
+	/// <summary>
+	/// Represents a model for file system operation progress.
+	/// </summary>
 	public class FileSystemProgress
 	{
-		private readonly IProgress<FileSystemProgress>? progress;
-		private readonly IntervalSampler sampler;
-		private FileSystemStatusCode? status;
-		private bool criticalReport;
-		private bool enumerationCompleted;
+		private readonly IProgress<FileSystemProgress>? _progress;
+
+		private readonly IntervalSampler _sampler;
+
+		private FileSystemStatusCode? _status;
+
```

Comment:
I think you rename this to `_Status` since it's the backing field of a public property. You should also not split it from its property

Reply:
Fixed